MediaRecorder.bs

```diff
@@ -286,11 +276,30 @@ interface MediaRecorder : EventTarget {
     <li>Set |recorder|'s {{state}} to {{recording}}, and run the following steps
     in parallel:
     <ol>
-      <li>Start recording all tracks in |tracks| using the {{MediaRecorder}}'s
-      current configuration, and gather the data into a {{Blob}} <var>blob</var>
-      and queue a task, using the DOM manipulation task source, to
-      <a>fire an event</a> named <a>start</a> at <var>recorder</var>, then return
-      <code>undefined</code>.</li>
+      <li>As soon as data is available on all tracks in |tracks|, the User Agent
+      MUST specify the |recorder|'s configuration for recording the tracks, if
+      not already specified. The User Agent MAY take the content of the first
+      frame into account when making this decision. The User Agent MUST then
+      start recording with this configuration and gather the data into a
+      {{Blob}} <var>blob</var>. As soon as recording starts, queue a task, using
+      the DOM manipulation task source, to run the following steps:
+      <ol>
+        <li>Set |recorder|'s {{mimeType}} to reflect the configuration,
+        including media type, subtype and codecs parameter. This MAY include the
+        profiles parameters [[RFC6381]] or further codec-specific
+        parameters.</li>
```

Comment:
Please keep the old language and algorithm for updating the mimeType. It was carefully worked out, I'd hate to see it go.